src/Windows/Containers/Test/WindowsContainerServiceTest.cs

```diff
@@ -64,29 +65,29 @@ public class WindowsContainerTests {
             await svc.StartContainerAsync(container, CancellationToken.None);
 
             var runningContainers = await svc.ListContainersAsync(false, CancellationToken.None);
-            runningContainers.Should().Contain(container.Id.Substring(0, 12));
+            runningContainers.Select((c) => c.Id).Should().Contain(container.Id);
 
             await svc.StopContainerAsync(container, CancellationToken.None);
 
             var runningContainers2 = await svc.ListContainersAsync(false, CancellationToken.None);
-            runningContainers2.Should().NotContain(container.Id.Substring(0, 12));
+            runningContainers2.Select((c) => c.Id).Should().NotContain(container.Id);
 
             await svc.DeleteContainerAsync(container, CancellationToken.None);
             var allContainers = await svc.ListContainersAsync(true, CancellationToken.None);
-            allContainers.Should().NotContain(container.Id.Substring(0, 12));
+            allContainers.Select((c) => c.Id).Should().NotContain(container.Id);
```

Comment:
If you want to see details about containers in test results, it is better to use `allContainers.Should().NotContain(c => c.Id == container.Id);`